composables/autoTeleport/useAutoTeleportActionButton.ts

```diff
@@ -0,0 +1,29 @@
+import type { Actions } from '@/composables/transaction/types'
+
+export default ({
+  autoTeleport,
+  autoTeleportButton,
+  autoTeleportLoaded,
+  getActionFn,
+}) => {
```

Comment:
nice! I was thinking if we could remove those params and have them created inside the composable, like action


that would remove some duplication between the two places where `useAutoTelportActionButton` is used 

wdyt, worth doing?


```suggestion
export default ({
  getActionFn,
}) => {
```